### PR TITLE
Save lidar calibration constants within att_bsc files

### PR DESCRIPTION
**Pull request by HolgerPollyNet**
added the storage of the used lidar constant  in netcdf files for att bsc and quasie results as attribute of the backscatters....typos fixed

### CODE REVIEW

File: lib/polly_general_func_lib/pollyxt_save_quasi_results.m [hunk 1]
```diff
@@ -124,6 +124,8 @@ function pollyxt_save_quasi_results(data, taskInfo, config)
 netcdf.putAtt(ncID, varID_quasi_bsc_532, 'source', campaignInfo.name);
 netcdf.putAtt(ncID, varID_quasi_bsc_532, 'error_variable', 'quasi_beta_532_error');
 netcdf.putAtt(ncID, varID_quasi_bsc_532, 'bias_variable', 'quasi_beta_532_bias');
+netcdf.putAtt(ncID, varID_quasi_bsc_532, 'Lidar_calibration_constant_used', data.LCUsed.LCUsed532);
+netcdf.putAtt(ncID, varID_quasi_bsc_532, 'Lidar_ratiot_used', config.LR532);
```
Comment (ZPYin) [inline lib/polly_general_func_lib/pollyxt_save_quasi_results.m:128]: please fix the typo here (`Lidar_ratiot_used` to `Lidar_ratio_used`)

File: lib/polly_general_func_lib/pollyxt_save_quasi_results.m [hunk 2]
```diff
@@ -137,6 +139,8 @@ function pollyxt_save_quasi_results(data, taskInfo, config)
 netcdf.putAtt(ncID, varID_quasi_bsc_1064, 'source', campaignInfo.name);
 netcdf.putAtt(ncID, varID_quasi_bsc_1064, 'error_variable', 'quasi_beta_1064_error');
 netcdf.putAtt(ncID, varID_quasi_bsc_1064, 'bias_variable', 'quasi_beta_1064_bias');
+netcdf.putAtt(ncID, varID_quasi_bsc_1064, 'Lidar_calibration_constant_used', data.LCUsed.LCUsed1064);
+netcdf.putAtt(ncID, varID_quasi_bsc_1064, 'Lidar_ratiot_used', config.LR1064);
```
Comment (ZPYin) [inline lib/polly_general_func_lib/pollyxt_save_quasi_results.m:143]: Same typo here.

**Review by ZPYin — CHANGES_REQUESTED** [review]
Please fix the typo marked by the comments and then send PR.

File: lib/polly_general_func_lib/pollyxt_save_quasi_results.m (hunk 1 above)
Comment (HolgerPollyNet) [reply lib/polly_general_func_lib/pollyxt_save_quasi_results.m]: I fixed the typos, thanks for the good reviewing

**Pull request merged**